src/style.css

```diff
@@ -0,0 +1,76 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  color: #fff;
+  font-size: 100px;
+}
+
+body {
+  margin: 0;
+  background-color: #ccc;
+}
+
+.conteiner {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.number {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-width: 300px;
+  height: 300px;
+  box-sizing: border-box;
+  flex-grow: 1;
+}
+
+.number-1 {
```

Comment:
flex order should be added to ALL flex children, otherwise it doesn't have sense